subscribers:

  camera_reading:
    topic: /usb_cam/image_raw
    queue_size: 1

publishers:

  control_data:
    topic: /twist_msg
    queue_size: 1

params:
  center_err: 510
  target_speed: 1.0
  angle_degree: 0.0
  speed_limits:
    max: 5.0
    min: 0.0

  angle_limits:
    max: 30.0
    min: -30.0

LaneDetector:
  
  sob_min_th: 175
  sob_max_th: 255
  his_min_th: 175
  hls_max_th: 255
  
  pid_params:
    Kp: 1.0
    Ki: 0.00001
    Kd: 0.0025
    dt: 0.1

image_view:

  enable_opencv: false
  wait_key_delay: 1
  enable_console_output: true
